projects/ziti-console-lib/src/lib/assets/scripts/chart.min.js

```diff
@@ -4,4 +4,4 @@
  * (c) 2019 Chart.js Contributors
```

Comment:
minified file changing is a 'code change' to me. also these third party libs, are you planning to keep them or remove them?

Reply:
Im not going to do a separate PR for removing an errant comment